integration/tests/cook/test_basic.py

```diff
@@ -306,6 +306,29 @@ def test_error_while_creating_job(self):
         resp = util.session.post('%s/rawscheduler' % self.cook_url,
                                  json={'jobs':[job1, job2]})
         self.assertEqual(resp.status_code, 500)
+    
+    def test_constraints(self):
+        # Want to ensure job is put on specific host, but no way to set host names in minimesos
```

Comment:
Can we add a test where we use the host name from `http://mesos-master:5050/state.json` user `slaves->*->hostname` ?

Then you can verify whether exactly that host was used. Maybe launch multiple jobs with different host constraints and verify each job only ran on the specified host?